dialects/linux/dsock.c

```diff
@@ -89,7 +87,7 @@ static char copyright[] =
 	+(int)TCPUDP6_IPC_ADDR_INT32(a, 0x3))
 
 #define TCPUDP6_IPC_HASH(tp) ((int)((((TCPUDP6_IPC_ADDR_MK_INT(&(tp)->faddr) \
-				       + TCPUDP6_IPC_ADDR_MK_INT(&(tp)->laddr), \
+				       + TCPUDP6_IPC_ADDR_MK_INT(&(tp)->laddr) \
 				       + (tp)->fport			\
 				       + (tp)->lport			\
 				       + (tp)->proto) * 31415) >> 3)	\
```

Comment:
Same remark as with the above UDP hash.

Reply:
Thank you.
However, in this pull request, I would like to focus on suppressing the warnings reported by the I compiler.
So, could you make a pull request after I merged this pull request?